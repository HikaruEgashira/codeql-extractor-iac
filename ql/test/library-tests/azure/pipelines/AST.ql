private import iac

query predicate adopipeline(AzurePipelines::Document n) { any() }

query predicate adopipelineVariable(AzurePipelines::Variable n) { any() }

query predicate adopipelineSteps(AzurePipelines::Step n) { any() }

query predicate adopipelinePool(AzurePipelines::Pool n) { any() }

query predicate adopipelineTask(AzurePipelines::Task n) { any() }

query predicate adopipelineScript(AzurePipelines::Script n) { any() }
