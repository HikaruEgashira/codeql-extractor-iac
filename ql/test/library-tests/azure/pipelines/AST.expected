adopipeline
| azure-pipelines.yml:1:1:29:26 | Azure DevOps Pipeline |
adopipelineSteps
| azure-pipelines.yml:16:5:21:2 | Azure DevOps Pipeline step |
| azure-pipelines.yml:21:5:26:2 | Azure DevOps Pipeline step |
| azure-pipelines.yml:26:5:29:26 | Azure DevOps Pipeline step |
adopipelinePool
| azure-pipelines.yml:5:3:5:25 | vmImage ... -latest |
adopipelineTask
| azure-pipelines.yml:16:5:21:2 | Azure DevOps Pipeline step |
adopipelineScript
| azure-pipelines.yml:21:5:26:2 | Azure DevOps Pipeline step |
| azure-pipelines.yml:26:5:29:26 | Azure DevOps Pipeline step |
